app/controllers/prove_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ProveController < ApplicationController
+  before_filter :check_write_permissions
+
+  def create
+    if ontology.unproven_theorems.present?
+      ontology.current_version.async_prove
+      flash[:success] = t('prove.create.starting_jobs')
+    else
+      flash[:notice] = t('prove.create.nothing_to_do')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.